contrib/node/src/python/pants/contrib/node/tasks/node_paths_local.py

```diff
@@ -0,0 +1,11 @@
+# coding=utf-8
+# Copyright 2015 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from pants.contrib.node.tasks.node_paths import NodePaths
+
+
+class NodePathsLocal(NodePaths):
+  """Maps Node package targets to their local NODE_PATH chroot."""
```

Comment:
The difference between `NodePathsLocal` and `NodePaths` needs more docs/comments... possibly in both the parent class and the subclass. It would be good to discuss why they are not the same class.

And in general, it's not a great idea to subclass concrete classes, as it makes the interactions between the parent and child harder to reason about.